src/CAN/can_mapping.c

```diff
@@ -32,6 +32,8 @@ float canmapping_extract_value(uint64_t raw_data, const CANMapping *mapping)
         }
         uint32_t bitmask = (1UL << length) - 1;
         uint32_t raw_value = (raw_data >> offset) & bitmask;
+
+        /* normalize endian */
```

Comment:
A runtime check for endian-ness would be useful. We can schedule this work if we ever switch to a different processor architecture.